fullcalendar/models.py

```diff
@@ -460,6 +460,35 @@ def format(self, tz, type='game-appointment'):
             })
         return event
 
+    def format_discord(self, tz=utc, locale=None):
+        """
+        Makes a Discord's embed object from event's data.
+        Optionnal timezone and locale can be used
+
+        """
+        date = timezone.localtime(self.start, tz).strftime('%d-%m-%Y %H:%M')
+        title = 'New game planned !'
+        divisions = []
+        if locale:
+            activate(locale)
+            title = str(_('Game Appointment Created Title'))
```

Comment:
why call `str`. `_()` return a str allready no?

Reply:
Because its a lazy loaded object
https://stackoverflow.com/questions/48454398/typeerror-at-en-object-of-type-proxy-is-not-json-serializable

Reply:
But I'm used gettext_lazy, I should try to use the non-lazy one

Reply:
fixed